components/basic/LoginDropdown.js

```diff
@@ -45,102 +41,85 @@ const DropdownCard = () => {
   };
 
   useEffect(() => {
-    const handleClickOutside = (event) => {
-      if (dropdownRef.current && !dropdownRef.current.contains(event.target)) {
-        setLoginOption(false);
-      }
-    };
-
-    document.addEventListener("mousedown", handleClickOutside);
+    document.addEventListener("click", handleOutsideClick);
     return () => {
-      document.removeEventListener("mousedown", handleClickOutside);
+      document.removeEventListener("click", handleOutsideClick);
     };
-  }, [dropdownRef]);
+  }, []);
 
   return (
-    <li>
-      <div className={styles.loginOption} ref={dropdownRef}>
-        {isUserLoggedIn || isMentorLoggedIn ? (
-          <button onClick={handleLoginClick} className={styles.loginbutton}>
-            <img
-              style={{
-                width: "35px",
-                height: "auto",
-                borderRadius: "50%",
-                display: "inline",
-              }}
-              src={
-                userData?.user_picture ||
-                mentorData?.mentor_picture ||
-                "assets/img/icon/no-profile-picture.webp"
-              }
-              alt="not found"
-            />
-          </button>
-        ) : (
-          <button
-            className={styles.loginbutton}
-            onClick={handleLoginClick}
-            style={{ color: "white" }}
-          >
-            Login
-          </button>
-        )}
+    <div ref={dropdownRef}>
+      {isUserLoggedIn || isMentorLoggedIn ? (
+        <button onClick={handleLoginClick}>
+          <img
+            style={{
+              width: "30px",
+              height: "auto",
+              marginRight: "10px",
+              borderRadius: "50%",
+              display: "inline",
+            }}
+            src={
+              localStorage.getItem("user_picture") ||
+              localStorage.getItem("mentor_picture") ||
```

Comment:
Actually we are using in a userdata and mentordata json.
See my previous comment

Reply:
Resolved

Reply:
I added the prev code as you mentioned..